tests/functional/test_signup_flow.py

```diff
@@ -37,4 +43,4 @@ def test_signup_flow(base_url, browser, outbox):
     assert logout_btn
     assert not logout_btn.visible
     submit_btn = browser.find_by_css('.dropdown-toggle')[0].click()
-    assert logout_btn.visible
+    assert logout_btn.visible"""
```

Comment:
@theskumar  The reason for commenting out test was not to merge it commented. As I have edited signup form, I was thinking to take your review on the form. If you agree to put those fields in signup form, I will complete this signup test case accordingly. Apologizing for not clearing that thing. Thanks